Dining-Philosophers-Tweello/tweello
# Tweel 38 implement edit user api endpoint

Created the API endpoint & functionality for edit user and get user profile. Also added the cookie generation generateToken function.

To test in Postman:

1. Find the user ID of the user you want to edit info for
2. In Postman, click on "Environments" --> "Tweello Auth"
3. Edit the Current Value for the variable "id" and press "save":
![image](https://github.com/Dining-Philosophers-Tweello/tweello/assets/59898052/d014fffc-7583-4c93-8cc4-9f04537ff00d)

4. Go back to Collections
5. Run the Edit User endpoint
Check MongoDB to confirm the data has changed
